manubot/cite/tests/test_citeproc.py

```diff
@@ -28,10 +36,12 @@ def test_remove_jsonschema_errors(name):
     To create a new test case, derive pruned.json from raw.json, by manually
     deleting any invalid fields. Do not use `manubot cite` to directly generate
     pruned.json as that also relies on remove_jsonschema_errors for pruning.
-    """
-    data_dir = directory / 'csl-json' / name
-    raw = json.loads(data_dir.joinpath('raw.json').read_text())
-    expected = json.loads(data_dir.joinpath('pruned.json').read_text())
+    """    
+    data_dir = directory / 'csl-json' / name 
+    def load_json(filename):
+        return json.load((data_dir / filename).open('r', encoding='utf-8'))
+    raw = load_json('raw.json')
+    expected = load_json('pruned.json')
```

Comment:
Any reason to change more than just adding `encoding='utf-8-sig'` as an arg to `.read_text`? I don't love defining this function to load a single json when the existing functions support everything needed

Reply:
> existing functions support everything needed
Actually there was no single function, but repeated code - one needs to copy-paste `encoding='utf-8-sig'` into several statements. 

Also `raw = json.loads(data_dir.joinpath('raw.json').read_text())` combines operations at different level of abstraction - it continues to specifiy the location of the file and the read operation. That makes for nested parenthesis, which are hard to read through quickly. Under these considerations I came to be thinking of `load_json(filename)`.